src/app/ui/bridge/StepDetail.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+import { Box, Typography } from '@mui/material';
+import Image from 'next/image';
+import type { ReactElement } from 'react';
+import { StepDetailContainer } from './StepDetail.style';
+
+const WIDGET_AMOUNT = 3;
+
+interface StepDetailImageProps {
+  imgUrl: string;
+  width: number;
+  height: number;
+  alt: string;
+}
+
+interface StepDetailProps {
+  title: string;
+  description: string;
+  img: StepDetailImageProps;
+  content?: ReactElement;
+}
+
+const StepDetail = ({ title, description, img, content }: StepDetailProps) => {
+  return (
+    <StepDetailContainer
+      sx={(theme) => ({ marginTop: theme.spacing(4) })}
+      imgWidth={img.width}
+    >
+      <Box>
+        <Typography
+          variant="h4"
+          marginY={2}
+          sx={{ fontSize: '24px!important' }}
```

Comment:
!important seems odd, is it mandatory?

Reply:
It was added by yourself when you first worked on that page - but deleting it did not seem to have any impact, so let´s get rid of those `!important`s

Reply:
Alright, after I removed them, it was obvious that those were actually needed -> The typography variants have some default media-queries which overwrite font-size on a certain breakpoint. For now, I reverted those changes and brought back all the `!important``s!

I´d suggest to clean this up in another round  @tcheee @oktapodia